index.html

```diff
@@ -1255,6 +1239,231 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `*_localized` members
+        </h3>
+        <p>
+          A <dfn>localizable member</dfn> is a [=manifest=] member that can be
+          localized. Each [=localizable member=] of the [=manifest=] has a
+          corresponding <code><dfn data-export="" data-dfn-for=
+          "manifest">*_localized</dfn></code> member, where `*` represents the
+          member name.
+        </p>
+        <aside class="example">
+          <p>
+            For example, the [=manifest/name=] member is a [=localizable
+            member=], and `name_localized` is its corresponding
+            [=manifest/*_localized=] member.
+          </p>
+        </aside>
+        <p>
+          A <dfn>language map</dfn> is an [=ordered map=] whose key is a
+          [=language tag=] and whose value is a [=localized value=]. The
+          <dfn>localized value</dfn> represents the value applied for the
+          [=language tag=] specified as the key.
+        </p>
+        <aside class="example" title="Localizing the application name">
+          <pre class="json">
+            {
+              "lang": "en-US",
+              "dir": "ltr",
+              "name": "Color Picker",
+              "name_localized": {
+                "de":    "Farbwähler",
+                "en":    {"value": "Color Picker"},
+                "en-GB": {"value": "Colour Picker", "dir": "ltr"},
+                "fr":    {"value": "Sélecteur de Couleur", "lang": "fr-CA", "dir": "ltr"},
+                "ar":    {"value": "منتقي الألوان", "dir": "rtl"}
+              }
+            }
+          </pre>
+        </aside>
+        <p>
+          [=manifest/*_localized=] members contain a [=language map=] that
+          defines [=localized values=] for the given [=localizable member=] in
+          the application. The user agent SHOULD use the user's localization
+          settings to select the [=localized value=] whose [=language tag=] key best
+          matches the user's preference. When no such [=localized value=] is
+          available, the default representation is used.
```

Comment:
"Default representation" should be defined (meaning the one at the root, if present).

Reply:
"Default representation" is already defined, and it refers to the member without the `*_localized` suffix.

It's a good question what should happen in this scenario:

```json
{
  // "name" (default representation) is missing
  "name_localized": {
    "de": "Guten Morgen",
    "fr": "Bonjour"
  }
}
```

We could simply ignore the `*_localized` member if there is no/an invalid default representation (case A).

Or, we could specify a fallback chain:
1. Match the `*_localized` against the user's locale settings and use the best match (case B).
2. If we have a valid `lang` specified at the root level, match the `*_localized` language map against it and use the best match (case C).
3. If there is still no match, ignore the member altogether (case D).

WDYT? @marcoscaceres @dmurph @aphillips 

<details>
<summary>Cases</summary>

## Case A

```json
{
  "name": 123, // invalid name
  "name_localized": {
    "de": "Guten Morgen",
    "fr": "Bonjour"
  }
}
```

| Locale | Result |
| --- | --- |
| `de` | — |
| `fr` | — |
| `en` | — |

## Case B

```json
{
  "name_localized": {
    "de": "Guten Morgen",
    "fr": "Bonjour"
  }
}
```

| Locale | Result |
| --- | --- |
| `de` | Guten Morgen |
| `fr` | Bonjour |
| `en` | — |

## Case C

```json
{
  "lang": "fr",
  "name_localized": {
    "de": "Guten Morgen",
    "fr": "Bonjour"
  }
}
```

| Locale | Result |
| --- | --- |
| `de` | Guten Morgen |
| `fr` | Bonjour |
| `en` | Bonjour |

## Case D

```json
{
  "lang": "en",
  "name_localized": {
    "de": "Guten Morgen",
    "fr": "Bonjour"
  }
}
```

| Locale | Result |
| --- | --- |
| `de` | Guten Morgen |
| `fr` | Bonjour |
| `en` | — |
</details>